app/lib/dartdoc/dartdoc_runner.dart

```diff
@@ -158,6 +158,12 @@ class DartdocJobProcessor extends JobProcessor {
     StringBuffer logFileOutput,
   ) async {
     logFileOutput.write('Running dartdoc:\n');
+    final canonicalVersion = job.isLatestStable ? 'latest' : job.packageVersion;
+    String canonicalUrl = pkgDocUrl(job.packageName,
```

Comment:
Wouldn't you want to fix `pkgDocUrl` here?

Or maybe use path.join?

Reply:
Added `omitTrailingSlash` to `pkgDocUrl`. Unfortunately, due to the dartdoc output's use of base href, we need to have a trailing slash when serving the content. We need to either add there (template output, /latest/ redirects) or remove here.